Makefile

```diff
@@ -47,11 +47,22 @@ db_rollback:
 db_seed:
 	docker-compose run ${RAILS_CONTAINER} rake db:seed
 
+.PHONY: schools_populate
```

Comment:
Pls revert all of these changes to the `Makefile`.  Reason being, they are one-off use cases that do not apply to recurring development.